src/UserInterface/CarMap.py

```diff
@@ -69,6 +70,11 @@ async def home_car_map():
 
         self.carMap_blueprint.add_url_rule("", "home_car_map", view_func=home_car_map)
 
+        @self._sio.on('window_loaded')
+        def __start_scoreboard_loop(sid):
+            self.__run_async_task(self.__emit_scoreboard())
```

Comment:
It would probably be good to check wether or not this loop is already running. If a second car map is opened, the loop would be started a second time wasting unnecessary resources, wouldn't it?